runtime/shibuya/src/lib.rs

```diff
@@ -1417,14 +1418,9 @@ impl pallet_treasury::Config<MainTreasuryInst> for Runtime {
     type Currency = Balances;
     type RuntimeEvent = RuntimeEvent;
 
-    // Two origins which can either approve or reject the spending proposal
```

Comment:
Regarding the removal of deprecated calls - won't the current changes completely break the treasury pallet?

If I'm not mistaken, the `SpendOrigin` needs to be updated to something that will actually work.

Reply:
That is correct. Good catch. Should be updated to use the correct origin